src/Interfaces/AbstractStar.jl

```diff
@@ -0,0 +1,9 @@
+"""
+    AbstractStar{N} <: LazySet{N}
+
+Abstract supertype for all star set types.
+"""
+abstract type AbstractStar{N} <: LazySet{N} end
```

Comment:
About the type hierarchy, `AbstractPolyhedron` could inherit from `AbstractStar`.